app/controllers/jobs_controller.rb

```diff
@@ -0,0 +1,68 @@
+class JobsController < ApplicationController
+  before_action :set_job, only: [:show, :edit, :update, :destroy]
+  before_action :set_company, only: [:new, :create, :show, :edit, :update, :destroy]
+
+  def index
+    if params.include?("sort")
+      @jobs = Job.all.order(:level_of_interest).reverse
+      render :interest
+    else
+      @company = Company.find(params[:company_id])
+      @jobs = @company.jobs
+      @contact = Contact.new
+      @contacts = @company.contacts.all.order('created_at DESC')
+    end
+  end
+
+  def new
+    @job = Job.new()
+    @category_options = Category.all.map{ |c| [c.title, c.id ]}
```

Comment:
this formatting piece could be done in a view helper instead since it has to deal with formatting something for our view!